Assignment03/Assignment03.css

```diff
@@ -0,0 +1,101 @@
+body{
+    margin: 0;
+    padding: 0;
+}
+#bgimg{
+    height:70vh; 
+    width: 100%;
+}
+.centered{
+    display: flex;
+    position: absolute;
+    top: 10%;
+    color: white;
+    left: 20vw;
+}
+.text{
+    width: 40vw;
+    margin-bottom:10vh;
+}
+.form{
+    width: 22vw;
+    height: 50vh;
```

Comment:
Was no need to write the height 50vh content can take default height

Reply:
Yes, Thank You. I made the change and its looking better now. Thanks once again.